docs/api_docs/kubeblocks-for-elasticsearch/manage-elasticsearch.md

```diff
@@ -0,0 +1,617 @@
+---
+title: Manage Elasticsearch with KubeBlocks
+description: How to manage Elasticsearch on KubeBlocks
+keywords: [elasticsearch]
+sidebar_position: 1
+sidebar_label: Manage Elasticsearch with KubeBlocks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Manage Elasticsearch with KubeBlocks
+
+## Before you start
+
+* [Install KubeBlocks](./../installation/install-kubeblocks.md).
+* View all the database types and versions available for creating a cluster.
+  
+  Make sure the `elasticsearch` cluster definition is installed. If the cluster definition is not available, refer to [this doc](./../overview/supported-addons.md#install-addons) to enable it first.
+
+  ```bash
+  kubectl get clusterdefinition elasticsearch
+  >
+  NAME                TOPOLOGIES   SERVICEREFS   STATUS      AGE
+  elasticsearch                                  Available   30m
+  ```
+
+  View all available versions for creating a cluster.
+
+  ```bash
+  kubectl get clusterversions -l clusterdefinition.kubeblocks.io/name=elasticsearch
+  ```
+
+* To keep things isolated, create a separate namespace called `demo` throughout this tutorial.
+
+  ```bash
+  kubectl create namespace demo
+  ```
+
+## Create a cluster
+
+KubeBlocks implements a `Cluster` CRD to define a cluster. Here is an example of creating a Qdrant Replication cluster. Primary and Secondary are distributed on different nodes by default. But if you only have one node for deploying a Replication Cluster, set `spec.affinity.topologyKeys` as `null`.
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: mycluster
+  namespace: demo
+spec:
+  clusterDefinitionRef: qdrant
+  clusterVersionRef: qdrant-1.8.1
```

Comment:
clusterVersionRef is deprecated, should use ComponentVersion, may ask @iziang to provide CR with new API.

Reply:
Fixed.